src/Command/EntryCommand.php

```diff
@@ -1,231 +0,0 @@
-<?php
-declare(strict_types=1);
-
-/**
- * CakePHP(tm) : Rapid Development Framework (http://cakephp.org)
- * Copyright (c) Cake Software Foundation, Inc. (http://cakefoundation.org)
- *
- * Licensed under The MIT License
- * For full copyright and license information, please see the LICENSE.txt
- * Redistributions of files must retain the above copyright notice.
- *
- * @copyright     Copyright (c) Cake Software Foundation, Inc. (http://cakefoundation.org)
- * @link          http://cakephp.org CakePHP(tm) Project
- * @since         2.0.0
- * @license       http://www.opensource.org/licenses/mit-license.php MIT License
- */
-namespace Bake\Command;
-
-use Bake\Shell\Task\BakeTask;
-use Cake\Command\Command;
-use Cake\Console\Arguments;
-use Cake\Console\Command\HelpCommand;
-use Cake\Console\CommandCollection;
-use Cake\Console\CommandCollectionAwareInterface;
-use Cake\Console\ConsoleIo;
-use Cake\Console\ConsoleOptionParser;
-use Cake\Console\Exception\ConsoleException;
-use Cake\Console\Shell;
-use Cake\Core\Configure;
-use Cake\Core\Plugin as CorePlugin;
-use Cake\Utility\Inflector;
-
-/**
- * Command that provides help and an entry point to bake tools.
- */
-class EntryCommand extends Command implements CommandCollectionAwareInterface
```

Comment:
What will be the output for `cake bake` if this command is removed?

Reply:
Well command's name doesn't really matter. We can just remove the unneeded parts of this command and avoid unnecessary code churn.

Reply:
Agreed.